test/performance/odp_crypto.c

```diff
@@ -80,6 +80,8 @@ static uint8_t test_key64[64] = { 0x01, 0x02, 0x03, 0x04, 0x05,
 typedef struct {
```

Comment:
A couple of typos in the commit description: "thew" and "used".

Reply:
Fixed in v2.